### ISSUE TITLE
display S statistic with P-value 

### ISSUE THREAD

**Issue by fairjane** [andrejadd/GWSDAT#273]
USEPA has requested the _S_ statistic in addition to p-values for MannKendall. It would be great to have them displayed with the p-value on the time series chart.

**Comment by Harold805** [comment]
Yes , the display of the S-value would be beneficial for our analyses.

**Comment by BHowell90** [comment]
Yes, this would be greatly beneficial for our analyses as well.

**Comment by courtneymcdavid** [comment]
I agree it would be useful information to have.

**Comment by fairjane** [comment]
fantastic, thank you! is there a release date for version 3.3?
